src/picongpu/include/simulation_defines/param/physicalConstants.param

```diff
@@ -27,6 +27,19 @@ namespace picongpu
 {
     BOOST_CONSTEXPR_OR_CONST float_64 PI = 3.141592653589793238462643383279502884197169399;
 
+    // Above this value (to the power of three, see comments on mapping) the synchrotron functions are nearly zero.
+    BOOST_CONSTEXPR_OR_CONST float_64 SYNC_FUNCS_CUTOFF = 5.0;
```

Comment:
IMO this should not part of this file, could you create an seperate one?
e.g. photon..., synchroton...